gcsio/src/main/java/com/google/cloud/hadoop/gcsio/authorization/StorageRequestAuthorizer.java

```diff
@@ -57,18 +57,19 @@ public StorageRequestAuthorizer(
    *
    * @param request Storage request to be authorized.
    * @throws AccessDeniedException Thrown when access denied by AuthorizationHandler.
+   * @return access token.
    */
-  public void authorize(StorageRequest<?> request) throws AccessDeniedException {
+  public String authorize(StorageRequest<?> request) throws AccessDeniedException {
```

Comment:
May be we could deprecate this method and re-factor to follow the new API, to use the same code path and delete the api in future release ?